src/index.ts

```diff
@@ -146,7 +146,7 @@ export class AwaitQueue
 		this.pendingTasks.length = 0;
 	}
 
-	dump(): { task: AwaitQueueTask;	name?: string; stopped: boolean }[]
+	dump(): { task: AwaitQueueTask<unknown>;	name?: string; stopped: boolean }[]
```

Comment:
Wrong format here, too many spaces.

Reply:
I don't know how that happened. Should fit now